storybook/stories/common/Widgets.stories.js

```diff
@@ -134,7 +134,7 @@ storiesOf('Common|Widgets', module)
     </div>
   ))
 
-  .add('BigButtonForDialogs', (props: { locale: string }) => (
+  .add('BigButtonForDialogs', (_, props: { locale: string }) => (
```

Comment:
Another breaking change:

```
In Storybook 6+, we pass the args as the first argument to the story function. The second argument is the “context”, which includes story parameters, globals, argTypes, and other information.

In Storybook 5 and before we passed the context as the first argument. If you’d like to revert to that functionality set the parameters.passArgsFirst parameter in .storybook/preview.js:

// .storybook/preview.js

export const parameter = { passArgsFirst : false }.
```

https://storybook.js.org/docs/react/writing-stories/args#mapping-to-complex-arg-values